src/index.html

```diff
@@ -2,12 +2,71 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+    <link
+      rel="stylesheet"
+      href="./styles/index.scss"
+    >
```

Comment:
make other in same way if you have less then 3 attribute
```suggestion
    <link rel="stylesheet" href="./styles/index.scss">
```